crates/ruff_formatter/src/format_element/document.rs

```diff
@@ -144,6 +144,12 @@ impl std::fmt::Display for DisplayDocument<'_> {
     }
 }
 
+impl std::fmt::Debug for DisplayDocument<'_> {
```

Comment:
Having `Debug` implementation can be useful when you want to use it with the `debug` macro.